### PR TITLE
return LoginRequired when not logged in for web-auth-token

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

File: go/service/main.go [hunk 1]
```diff
@@ -1439,3 +1439,11 @@ func (d *Service) StartStandaloneChat(g *libkb.GlobalContext) error {
 
 	return nil
 }
+
+func assertLoggedIn(ctx context.Context, g *libkb.GlobalContext) error {
```
Comment (zapu) [inline go/service/main.go]: there are two `assertLoggedIn` in `teams.go` and ` kvstore.go`, I wonder if we can move them to BaseHandler and use it instead

Comment (heronhaye) [reply go/service/main.go]: basehandler is not contextified. but I can refator teams and kvstore to use this fn

Comment (zapu) [reply go/service/main.go]: sounds good, thank you!

Comment (heronhaye) [reply go/service/main.go]: donezo

**Pull request merged**